Kubernetes/flannel/start.ps1

```diff
@@ -0,0 +1,75 @@
+﻿Param(
+    [ValidateSet("l2bridge", "overlay",IgnoreCase = $true)] [parameter(Mandatory = $true)] $NetworkMode,
+    [parameter(Mandatory = $false)] $ClusterCIDR="10.244.0.0/16",
+    [parameter(Mandatory = $false)] $KubeDnsServiceIP="10.96.0.10",
+    [parameter(Mandatory = $false)] $ServiceCIDR="10.96.0.0/12",
+    [parameter(Mandatory = $false)] $InterfaceName="Ethernet",
+    [parameter(Mandatory = $false)] $LogDir = "C:\k",
+    [parameter(Mandatory = $true)] $ManagementIP
```

Comment:
So, we're not going to have $IsolationType anymore?

Reply:
Yes-- I'm curious about this also. What about support for the `-feature-gates=HyperVContainer=true` flag?

Reply:
Are you using this and does it work for you? This was originally added purely for consistency with the other start.ps1 scripts in this repo, but there was some issues with endpoint attach since it was in alpha. These scripts are intended for customers to be a reference to get started, then they can customize it to their liking by adding flags for custom alpha features like DSR or hyper-V pods which should be documented separately.

The only exception for this right now is the overlay feature, which is a network mode we want customers to be able to select easily vs. l2bridge.